index.html

```diff
@@ -3482,7 +3480,7 @@ <h3>Proof Formats</h3>
 
         <ul>
           <li>
-Verifiable Credentials using JSON Web Tokens [[?VC-JWT]], and
```

Comment:
@msporny what is the reason for this change?

Reply:
To normatively reference VC-JWT a proof mechanism.